packages/site-kit/src/lib/docs/DocsOnThisPage.svelte

```diff
@@ -141,30 +153,50 @@
 />
 
 <aside class="on-this-page" bind:this={containerEl}>
-	<h2>On this page</h2>
-	<nav aria-label="On this page">
-		<ul>
-			<li>
-				<a
-					href="{base}/docs/{details.slug}"
-					class:active={hash === ''}
-					on:click={() => dispatch('select')}>{details.title}</a
-				>
-			</li>
-			{#each details.sections as { title, slug }}
+	<button class="heading" on:click={() => (mobile_menu_open = !mobile_menu_open)}>
```

Comment:
does this button only do something when on a mobile screen?

if so, we should only render a button when we're on a mobile screen. otherwise we'll have a control that does nothing at some screen widths

Reply:
On on mobile screen, yep!

Is rendering separate elements necessary? That would complicate things somewhat, and I'd rather not do it, unless its a deal breaker for screen readers?

Reply:
It's not great, since you have an interactive control that doesn't do anything. But as a screen reader user you might assume that it is doing something, it's just not being announced properly

Can't we just use `<svelte:element>` and toggle between button and div?

Reply:
Done